src/concepts/pipeline/data_pipeline_detail.md

```diff
@@ -186,7 +186,7 @@ Spark Streaming is used to read from Kafka and perform [low-latency ETL and aggr
 
```

Comment:
This is an obsolete guide to the old AWS data pipeline, so not worth fretting too much about losing links.